api/goldens/aptos_api__tests__transactions_test__test_get_transactions_returns_last_page_when_start_version_is_not_specified.json

```diff
@@ -2065,7 +2065,7 @@
     "state_change_hash": "",
     "event_root_hash": "",
     "state_checkpoint_hash": null,
-    "gas_used": "1538",
+    "gas_used": "1537",
```

Comment:
Why did this decrease?

Reply:
New compiler optimizations: https://github.com/move-language/move/commit/f04e00267dfb03c9efb11fb1b768c1cbc1341b8b

Reply:
Lol saving of 1 gas unit 😄